recipes/cpp_comment_format/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "cpp_comment_format" %}
+{% set org = "tdegeus" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ org }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 888bfff303dfbd9aee16a0ba6d5c266fe24e319c4c91975fd0a8f16220f3176f
+
+build:
+  number: 0
+  noarch: python
+  script_env: SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - {{ name }} = {{ name }}:_cli
+
+requirements:
+  host:
+    - pip
+    - python >=3
+    - setuptools_scm
+  run:
+    - python >=3
+
+test:
+  requires:
+    - pip
+  imports:
+    - {{ name }}
+  commands:
+    - pip check
+    - python -c "import {{ name }}; assert {{ name }}.version == '{{ version }}'"
+    - {{ name }} --version | grep {{ version }}
+
+about:
+  home: https://github.com/{{ org }}/{{ name }}
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: Format C++ comment blocks.
+  description: Format C++ comment blocks.
+  doc_url: https://{{ name }}.readthedocs.io/
```

Comment:
This url doesn't seem to exist.

Personal opinion, but I think the variables `name` and `org` everywhere make the recipe more difficult to read.

Otherwise, looks good!

Reply:
@beenje Thanks! The build on readthedocs was failing. Good to go now!